pkg/command/install/uninstall.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Singularity Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package install
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/singularity-data/risingwave-operator/pkg/command/context"
+	"github.com/singularity-data/risingwave-operator/pkg/command/util"
+)
+
+const (
+	UninstallExample = `  # uninstall the risingwave operator from the cluster
+  kubectl rw uninstall
+`
+)
+
+// UninstallOptions contains the options for the uninstallation command.
+type UninstallOptions struct {
+	genericclioptions.IOStreams
+}
+
+// NewUninstallOptions returns a UninstallOptions.
+func NewUninstallOptions(streams genericclioptions.IOStreams) *UninstallOptions {
+	return &UninstallOptions{
+		IOStreams: streams,
+	}
+}
+
+// NewUninstallCommand creates the uninstallation command of the operator.
+func NewUninstallCommand(ctx *context.RWContext, streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewUninstallOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:     "uninstall",
+		Short:   "Uninstall the risingwave operator from the cluster",
+		Long:    "Uninstall the risingwave operator from the cluster",
+		Example: UninstallExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			util.CheckErr(o.Complete(ctx, cmd, args))
+			util.CheckErr(o.Run(ctx, cmd, args))
+		},
+	}
+
+	return cmd
+}
+
+func (o *UninstallOptions) Complete(ctx *context.RWContext, cmd *cobra.Command, args []string) error {
+	return nil
+}
+
+// Run will run the command as followed:
+// 1. ensure the latest risingwave file exists.
+// 2. delete the resource by the risingwave file.
+func (o *UninstallOptions) Run(ctx *context.RWContext, cmd *cobra.Command, args []string) error {
+	yamlFile, err := Download(RisingWaveUrl, TemDir+"/operator")
```

Comment:
It's possible that the user have installed a previous version but delete the latest version. There could be mismatch between the resources, and will leak resources. Should we create some configurations inside the namespace to tell what version is installed? So that we can precisely delete with the same file.

Furthermore, should we check if there're RisingWaves running in the Kubernetes and give a notice if true before proceeding to the deletion?

Reply:
This is a big change and I created an issue to track the fix. 
https://github.com/singularity-data/risingwave-operator/issues/184